cmd/whitesourceExecuteScan.go

```diff
@@ -366,29 +665,29 @@ func autoGenerateWhitesourceConfig(config *ScanOptions, cmd *command.Command) er
 	}
 
 	// archiveExtractionDepth=0
-	if err := cmd.RunExecutable("sed", "-ir", `s/^[#]*\s*archiveExtractionDepth=.*/archiveExtractionDepth=0/`,
+	if err := utils.RunExecutable("sed", "-ir", `s/^[#]*\s*archiveExtractionDepth=.*/archiveExtractionDepth=0/`,
```

Comment:
Not introduced in this PR, but do we really need/want to depend on `sed`? this will make the piper binary less portable, even probably between linux versions with different implementations of sed, let alone macos or windows. Do we really need this? I assume the same should be possible using native go code.

Reply:
For what it is worth, I remember thinking the same thing when I came across this. Totally agree.